Source/AlchemyDataNewsV1/Models/ImageKeyword.swift

```diff
@@ -17,21 +17,21 @@
 import Foundation
 
 /**
- 
+
```

Comment:
Do you need to leave a newline for every line in the documentation?

Reply:
You're right, this is an odd format for the documentation. I think this code/documentation is pretty old--probably before we had any consistent style for comments, maybe even before we started generating documentation with Jazzy.

I think we may want to drop some of these older retired services in the next major release. If that's the case, then it may not be worth the time to update this documentation right now.

Reply:
I can't comment on your review itself, so I'll have to put my message here:

Nice find on the `w=1` option for the GitHub diff! Didn't know you could do that. That's very handy.